stdlib/public/core/Unicode.swift

```diff
@@ -140,260 +227,61 @@ public struct UTF8 : UnicodeCodec {
     let bit1 = (lut1 >> index) & 1
 
     switch (bit1, bit0) {
-    case (0, 0): // 2-byte sequence.
+    case (0, 0): // 2-byte sequence, [ XXX XXX CU1 CU0 ].
```

Comment:
I don't know; I can't tell what you're trying to express here.  presumably CU1 and CU0 are additional bytes, so it looks like a 4-byte sequence.